frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -123,13 +123,19 @@ object FrontendApp extends JSApp with Persistence with PrettyJson with Manipulat
     handleDownstream(messagesPanel.messageReceived)
   ).connect()
 
+  private def openDetailsInTab(actorRef: String): Unit = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [openDetailsInTab should have a smaller scope by using private[this]](https://www.codacy.com/app/lustefaniak/akka-viz/file/2204224192/issues/source?bid=3121282&fileBranchId=3170129#l126)